src/Microsoft.TemplateSearch.Common/TemplateSearchCache/TemplateSearchCache.cs

```diff
@@ -12,12 +14,14 @@ internal partial class TemplateSearchCache
 
         internal TemplateSearchCache(IReadOnlyList<TemplatePackageSearchData> data)
         {
-            TemplatePackages = data;
+            // when creating from freshly-generated data, order the results for clarity
```

Comment:
This change should make it easier for us to do side-by-side compares in the future. When I did this, it made it very easy to see that most diffs were like this:

![image](https://github.com/dotnet/templating/assets/573979/8766ef50-6617-482f-b8b3-7c65294d5d82)

meaning descriptions, versions, download counts that had changed over time. The other big source of diffs was additions/removals of entire packages from the cache file, which is inevitable until we move to searching the entire NuGet feed.